_test/helper.test.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+class helper_plugin_captcha_public extends helper_plugin_captcha {
+
+    public function get_field_in() {
+        return $this->field_in;
+    }
+    public function get_field_sec() {
+        return $this->field_sec;
+    }
+    public function get_field_hp() {
+        return $this->field_hp;
+    }
+}
+
+/**
+ * @group plugin_captcha
+ * @group plugins
+ */
+class helper_plugin_captcha_test extends DokuWikiTest {
+
+    protected $pluginsEnabled = array('captcha');
+
+    public function testConfig() {
+        global $conf;
+        $conf['plugin']['captcha']['lettercount'] = 20;
+
+        $helper = new helper_plugin_captcha_public();
+
+        $code = $helper->_generateCAPTCHA("fixed", 0);
+        $this->assertEquals(16, strlen($code));
```

Comment:
I think it's fine. We might want to add a maximum check to the config setting. In general I doubt anyone wants to use more than 16 characters. It would be a PITA to type that.
If we ever add a captcha mode where more chars are required we can always extend this.